meilisearch/_httprequests.py

```diff
@@ -64,7 +65,7 @@ def send_request(
                 serialize_body = isinstance(body, dict) or body
                 data = (
                     json.dumps(body, cls=serializer)
-                    if serialize_body
+                    if isinstance(body, bool) or serialize_body
```

Comment:
I believe these nested `if`s are bit convoluted. But I decided not revamping them now :) Previous PRs suggest it might be for backward compatibility